### PR TITLE
LRQA-82599

### PR DESCRIPTION
**Pull request by HenriiqueSantos**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:11]: Please check : https://github.com/lucasperj/liferay-portal/pull/372/files#r1315969960

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Age",
+				label = "Idade");
+		}
+
+		task ("We publish the form, save the form, and navigate to the published form") {
+			Form.publishForm();
+
+			Form.save();
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:62]: Please check : https://github.com/lucasperj/liferay-portal/pull/372/files#r1315997906

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Henrique",
+				key_rowEntry = 23);
+
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Jose Henrique Santos de lima",
+				key_rowEntry = 23);
+		}
+	}
+
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:119]: - Please check : https://github.com/lucasperj/liferay-portal/pull/372#issuecomment-1706815755

Finally, what we talked about on Friday, the ideal here would be for you to either edit the existing commit to change the wrong message OR reset it to the upstream hash after the rebase and commit again, but not put the other commit on top with just the changes and practically the same commit message, in the commit history the commit with the wrong message still appears and makes it more confusing when reviewing, please fix that.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+		task ("Aentering values into the form and sending the entered values") {
+			FormViewBuilder.editText(
+				fieldName = "Full Name",
+				fieldValue = "Henrique");
+
+			FormViewBuilder.editText(
+				fieldName = "Age",
+				fieldValue = 23);
+
+			FormPortlet.submit();
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:76]: - Use just FormPortlet.submitSuccessfully();, this way you ensure that the form has been submitted

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by lucasperj** [comment]
@HenriiqueSantos 

- Still need to fix the commit message of [66d77](https://github.com/lucasperj/liferay-portal/pull/375/commits/16de9c0580afb46bd7b0dd2f28e0da1f93e66d77)
- Remember to change the post commit message of [0fa37](https://github.com/lucasperj/liferay-portal/pull/375/commits/d4a303085d288904effb08575cde44d3eff0fa37) only mentioning the changes that were made
- A different and maybe easy solution would be, as suggested earlier, to reset to the upstream hash after rebasing and committing again. Please fix it. Any questions send me a message on slack

**Comment by HenriiqueSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 5]
```diff
@@ -0,0 +1,126 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:10]: :yellow_circle: Empty line, as we talked some minutes ago, you need to execute SF after make changes

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+			FormViewBuilder.editText(
+				fieldName = "Nome Completo",
+				fieldValue = "Lucas Falcao");
+
+			FormViewBuilder.editText(
+				fieldName = "Idade",
+				fieldValue = 55);
+				
+			FormPortlet.submitSuccessfully();
+
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:101]: :yellow_circle: Empty line, as we talked some minutes ago, you need to execute SF after make changes

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+			FormsAdminNavigator.gotoViewEntries();
+
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Lucas Falcao",
+				key_rowEntry = 55);
+
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Jose Henrique Santos de lima",
+				key_rowEntry = 23);
+
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase:122]: :yellow_circle: Empty line, as we talked some minutes ago, you need to execute SF after make changes

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by lucasperj** [comment]
Looks good to me :rocket:

**Pull request closed without merging**